test/ig/IGExporter.test.ts

```diff
@@ -485,6 +485,33 @@ describe('IGExporter', () => {
     });
   });
 
+  describe('#customized-ig', () => {
```

Comment:
Could you change this name? There is already a `customized-ig` describe block.

Reply:
Whoops.  Missed that one.  Fixed in 2353baa.